management/management_option.go

```diff
@@ -77,16 +77,6 @@ func WithClient(client *http.Client) Option {
 	}
 }
 
-// WithAuth0ClientInfo configures the management client to use the provided client information
-// instead of the default one.
-func WithAuth0ClientInfo(auth0ClientInfo client.Auth0ClientInfo) Option {
```

Comment:
We don't want to adjust this one too?

Reply:
I figured that given it's existed for a few releases and has never been noticed til we noticed the retryconfig issue then it isn't really needed. Really the only pieces needed are adding to env info and removing the client info from being sent which `WithAuth0ClientEnvEntry` and `WithNoAuth0ClientInfo` achieve